wvd-sh/WVD scaling script/basicScale-MSI.ps1

```diff
@@ -691,16 +719,26 @@ else {
                 #check whether the number of running VMs meets the minimum or not
                 if ($numberOfRunningHost -lt $MinimumNumberOfRDSH) {
                     $VMName = $sessionHost.Split(".")[0]
+                     $VmInfo = Get-AzureRmVM | Where-Object { $_.Name -eq $VMName }
+                    #Check the Session host is in Maintainance
+                    if($VmInfo.Tags.Keys -contains $maintainanceTagName){
+                        Write-Log 1 "Session Host is in Maintainace: $($sessionhost | out-string )"
```

Comment:
Replace "Maintainace" with "maintenance"

Reply:
We have replaced in both scripts